Docs: Add basic application tutorial (#1515)
litestar-org/litestar: docs/tutorials/todo-app/3-assembling-the-app.rst

Recap and assembling the final application
===========================================

So far we have looked at the different parts of the application in isolation, but now

> it's time to put them all together an assemble a fully working application from them.

Reviewer: ```suggestion
it's time to put them all together and assemble a complete application.
```
